schema.json

```diff
@@ -186,6 +186,37 @@
         },
         "epoch": {
           "type": "integer"
+        },
+        "syscalls": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/syscall"
+          }
+        }
+      }
+    },
+    "randomness": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/base64"
+      }
+    },
+    "syscall": {
```

Comment:
randomness may get used in some implicit contexts as well, which is why it's split out, it also should make sense that it's immutable for a requested epoch, rather than being memoized per request.

For the expected syscalls, we currently only stub out a minimal set in the lotus driver, but it seems like the full lotus runs with a larger set. I imagine there is a full enumeration of that expected surface area somewhere?

verification of order and calls seems like a thing the driver can/should do.